code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -726,9 +724,8 @@
 	ph = 2
 	purity = REAGENT_STANDARD_PURITY
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
-	impure_chem = /datum/reagent/impurity/inacusiate
 	inverse_chem_val = 0.3
-	inverse_chem = /datum/reagent/impurity/inacusiate
+	inverse_chem = /datum/reagent/impurity/inacusiate //I think this can stay
```

Comment:
```suggestion
	inverse_chem = /datum/reagent/impurity/inacusiate
```

Comments should describe what the code is now, not what it used to be.

Reply:
Change made!